packages/api/src/lib/granule-delete.ts

```diff
@@ -128,13 +95,6 @@ const deleteGranuleAndFiles = async (params: {
       await granulePgModel.delete(trx, {
```

Comment:
await createRejectableTransaction can be removed since there is only one sql statement.